### ISSUE TITLE
Consider adding a License to the code so others can reuse it

### ISSUE THREAD

**Issue by asayler** [untra/cufcq#1]
See http://choosealicense.com/ for some ideas.

If you want a standard copyleft-license, the AGPL is often a good fit for web-based projects like this since it requires anyone who hosts a copy of the site to make the source code available, something more traditional licenses lack. See http://www.gnu.org/licenses/why-affero-gpl.html.

**Comment by untra** [comment]
Thanks for the advise Andy! Except this is actually the old repository for our code. The current repository can be found here: https://github.com/antsankov/cufcq_dev

Nonetheless we will get a good licence attached. Cheers!
-Samuel Volin

**Comment by asayler** [comment]
The new repo doesn't have a public bug tracker. If you enable that (in the github project settings), I can post there.

**Comment by antsankov** [comment]
Hi, Andy. We've done a couple of things to address this issue: 

- We just created a new git user, and [repo fork](https://github.com/cufcq/cufcq) that we will be working off of in the future.

- We changed all of the links on the website to point to this new repo. 

- We've selected the MIT license. This seems to be pretty permissive and will allow us to stay flexible in terms of future changes. 

If there is any other feedback/advice you have about how everything is organized or licensing, we would really appreciate it! Thanks for commenting.
